### ISSUE TITLE
img alt text is invalid if it is not specified in source file

### ISSUE THREAD

**Issue by erfaan** [getpelican/pelican#647]
img alt text is invalid if no alt text is specified in source file. This is happening after new `_update_content`

Previously it was

```
static/pictures/Sushi.jpg
```

Now it is

```
|filename|/pictures/Sushi.jpg
```

Ideally, it should degrade gracefully and should be same as `src` of the image if not specified.
